msteams-platform/resources/teams-updates.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Microsoft Teams updates
+description: Learn about the latest updates to Microsoft Teams.
+author: v-ypalikila
+ms.author: lajanuar
+ms.localizationpriority: medium
+ms.topic: reference
+---
+# Microsoft Teams update
+
+[New Microsoft Teams client](https://www.microsoft.com/microsoft-365/blog/2023/03/27/welcome-to-the-new-era-of-microsoft-teams/) is reimagined from the ground up with performance in mind. It's faster, simpler, smarter, and flexible to provide better experience for your apps and users. The new Teams Client is closely aligned with all the existing Teams Clients (desktop, web and mobile) where your apps run. You can use the new Teams Client with all the existing capabilities in Teams at full feature parity.
+
+If you have an app that runs inside the Classic Teams Client for Windows or Mac, Teams Web Clients for Edge, Chrome, or Firefox, and Teams Mobile Clients for Android and iOS, the app will most likely run in the new Teams Client without any issues.
+
+Apps in the new Teams client run better and faster as:  
+
+* The new Teams Client uses the Evergreen version of Edge WebView2 to ensure Teams client is always up to date with the latest fixes and improvements available in Edge and Chromium.
+
+* The new Teams Client has been rebuilt from the ground up with performance in mind and includes all the platform infrastructure responsible for bootstrapping your app and powering the SDK APIs that it uses.  
+
+## Timelines and rollout
+
+To ensure a smooth transition, a phased rollout of the new platform is planned as follows:
+
+* **Developer Preview**: The new Teams client is available in Public Developer Preview starting 01 June 2023. You can access the new platform and test your apps. We encourage you to adopt the feature early and provide feedback to help refine the platform.
+
+* **Availability of all platform features from Teams classic**: All the platform features from Teams classic will be available in the new Teams client by August 2023. We plan to officially launch the new platform after addressing the known issues and incorporating feedback.
```

Comment:
I see many different ways in which we are referring to the Teams Clients; should we clean this up (at least the capitalization)?
- New Teams Client
- new Teams Client
- new Teams client
- Teams classic
- Classic Teams Client
- classic Teams client